src/pages/organizations/[organizationid].js

```diff
@@ -463,7 +463,7 @@ export default function Organization(props) {
               mt: [10, 20],
             }}
           />
-          <ImpactSection />
+          {/**<ImpactSection />  */}
```

Comment:
@Veenakrishna2313 please simply delete the code, a clear codebase is more important than considering some reuse of code in the future.

Reply:
I have deleted the code